frontend/app/src/router/Main.tsx

```diff
@@ -1,17 +1,66 @@
 import { Route, Routes } from 'react-router-dom'
 import Nav from './Nav'
-import { GameView, FriendView, ChatView, ProfileView } from 'view'
+import { GameView, FriendView, ChatView } from 'view'
+import { Profile } from 'components/profile/Profile'
+
+import { mockUser } from 'mock/mockUser'
+import { useEffect, createContext, useState } from 'react'
+import { io, Socket } from 'socket.io-client'
+
+export const SOCKET_EVENT = {
+  JOIN_ROOM: 'JOIN',
+  SEND_MESSAGE: 'SEND',
+  RECEIVE_MESSAGE: 'RECEIVE',
+  NOICE: 'NOTICE',
+  CREATE: 'CREATE',
+}
+
+type Data = {
+  roomId: number
+  senderUid: number
+  msgContent: string
+}
+
+// interface ServerToClientEvents {
+//   noArg: () => void
+//   basicEmit: (a: number, b: string, c: Buffer) => void
+//   withAck: (d: string, callback: (e: number) => void) => void
+// }
+
+// interface ClientToServerEvents {
+//   hello: () => void
+// }
```

Comment:
```suggestion
import { Message, ClientToServerEvents, ServerToClientEvents } from 'data'
```
[frontend/app/src/data](https://github.com/exciting-transcendence/transcendence/blob/1d2f8184c58e753d439fa1164f14b5f9a6634e26/frontend/app/src/data/Chat.dto.ts) 에 관련 정보들을 담아뒀어요